src/view/TestRunner.java

```diff
@@ -0,0 +1,40 @@
+import javax.swing.JFrame;
+
+/**
+ * Form to show a question and navaigation buttion and to give user
+ * functionality to select one of the options.
+ * @author      Sarthak Tiwari <sarthak.tiwari@asu.edu>
+ * @version     1.0
+ */
+public class TestRunner {
+
+    public static void main(String[] args) {
+
+        JFrame f = new JFrame();
```

Comment:
let's replace 'f' with something meaningful.

Reply:
Test Runner class won't be part of the deliverable, that is just a sample code showing how to use the class.

Reply:
Does 'reponseStatus' sound okay ?

Reply:
ok